backend/src/emissions/schema.py

```diff
@@ -964,7 +969,51 @@ def mutate(root, info, input):
             return RemoveUserFromWorkingGroup(success=True)
         except ValidationError as e:
             return RemoveUserFromWorkingGroup(success=False, errors=e)
+        
+        
+class AddUserToWorkingGroup(graphene.Mutation):
+    """GraphQL mutaiton to add a user to a working group"""
     
+    class Arguments:
+        """Input strucutred defined by input type"""
+        
+        input = AddUserToWorkingGroupInput()
+        
+    success= graphene.Boolean()
+        
+    @staticmethod
+    @login_required
+    @representative_required
+    def mutate(root, info, input):
+        """Process incoming data"""
+        
+        user = info.context.user
+        
+        working_group = WorkingGroup.objects.filter(id=user.working_group.id)[0]
+        
+        if working_group is None:
+            raise GraphQLError("You are not part of a working group. You can only add users to your own working group.")
+        if info.context.user.id != working_group.representative.id:
+            raise GraphQLError("You are not the representative of the specified working group. Unable to delete")
+
+        user_to_add_query_set = CustomUser.objects.filter(email=input.user_email)
+        
+        if len(user_to_add_query_set) > 1:
+            raise GraphQLError(f"Invalid email adress input. Multiple users found.") # this should never happen as email addresses should be unique in the database
+        elif len(user_to_add_query_set) == 0:
+            raise GraphQLError(f"There is no user for your provided input. Please try another email.")
```

Comment:
Maybe a more user-friendly message like "The user that you are trying to add is not registered yet. "?